kythe/cxx/common/path_utils.cc

```diff
@@ -132,14 +132,27 @@ absl::string_view Dirname(absl::string_view path) {
   return SplitPath(path).dir;
 }
 
+namespace {
+bool IsProperPathPrefix(absl::string_view s, absl::string_view prefix) {
+  return absl::StartsWith(s, prefix) &&
+         (s.size() == prefix.size() || s[prefix.size()] == '/');
+}
+}  // namespace
+
```

Comment:
Put this in the extant anonymous namespace rather than introducing another to put it here.

Reply:
Done.